src/configure-karma/index.js

```diff
@@ -26,7 +26,24 @@ const buildStandardKarmaConfig = (saguiConfig, webpackConfig) => {
       ...(coverage ? ['coverage'] : [])
     ],
 
-    frameworks: ['jasmine'],
+    frameworks: ['jasmine', 'detectBrowsers'],
+
+    detectBrowsers: {
+      enabled: additionalKarmaConfig.browsers && additionalKarmaConfig.browsers > 0,
+      usePhantomJS: true,
+
+      postDetection: function (availableBrowser) {
+        const isTravis = process.env['TRAVIS'] === 'true'
+        const isChromeAvailable = availableBrowser.indexOf('Chrome') > -1
+        const isTravisChromeAvailable = (process.env['TRAVIS_STACK_FEATURES'] || '').indexOf('google-chrome') !== -1
+
+        if (isChromeAvailable && (!isTravis || (isTravis && isTravisChromeAvailable))) {
```

Comment:
Me and @npejo just paired on this solution, a custom detection to work-around Travis.